entregas/lopezAndres/reto-001/DevolucionDinero.java

```diff
@@ -0,0 +1,37 @@
+
+public class DevolucionDinero {
+    public static void main(String [] args) {
+		int dineroPagado = 1000;
+		int dineroAPagar = 746;
+		int cambio = dineroPagado - dineroAPagar;
+		
+		int billetes100 = (int) cambio / 100;
```

Comment:
Dado que cambio es entero, el casting no es necesario

```suggestion
		int billetes100 = cambio / 100;
```